openedx/adg/lms/applications/tests/test_serializers.py

```diff
@@ -89,24 +86,29 @@ def test_education_serializer_with_valid_data(education_data):  # pylint: disabl
     assert not serializer.errors
 
 
+@pytest.mark.parametrize(
+    'degree, area_of_study, expected_area_of_study', [
+        (Education.HIGH_SCHOOL_DIPLOMA, 'Computer Sciences', ''),
+        (Education.HIGH_SCHOOL_DIPLOMA, '', ''),
+        (Education.BACHELOR_DEGREE, 'Computer Sciences', 'Computer Sciences')
+    ]
+)
 @pytest.mark.django_db
-def test_education_serializer_with_valid_degree_data(education_data):  # pylint: disable=redefined-outer-name
+def test_education_serializer_with_valid_degree_data(
+    degree, area_of_study, expected_area_of_study, education_data
+):  # pylint: disable=redefined-outer-name
```

Comment:
since this warning is suppressed in many instances within the module, let's please remove it from all instances within the module and move it to the top